### PR TITLE
Updated de & fr

### PR DESCRIPTION
**Pull request by MacSteini**
Updated de

Made a fundamental mistake when I ran an automated script and messed up quota/lang/fr

### CODE REVIEW

File: file/lang/de [hunk 1]
```diff
@@ -1,6 +1,6 @@
 index_title=Java-Dateimanager
 index_nojava=Dieses Modul benötigt Java, um zu funktionieren, aber Ihr Browser unterstützt Java nicht.
-index_eremote=Es gibt keine:n Unix-Benutzer:in, der dem Webmin-Login $1 entspricht.
+index_eremote=Es gibt keine:n Unix-Benutzer:in, der/die dem Webmin-Login $1 entspricht.
```
Comment (iliajie) [inline file/lang/de:3]: Thanks! Though, isn’t there a more inclusive way to phrase this? What about using `xier` instead of the cumbersome `der/die`?

Comment (MacSteini) [reply file/lang/de:3]: This is the common and established way it is being used in German language.

Comment (iliajie) [reply file/lang/de:3]: Alright!

**Comment by iliajie** [comment]
Thanks for the patches!

**Pull request merged**